ecosystem/indexer-grpc/indexer-grpc-data-service/README.md

```diff
@@ -15,19 +15,36 @@ Indexer GRPC data service fetches data from both cache and file store.
 ```yaml
 health_check_port: 8083
 server_config:
-    data_service_grpc_listen_address: 0.0.0.0:50052
     whitelisted_auth_tokens: 
       - "token1"
       - "token2"
     file_store_config:
       file_store_type: GcsFileStore
       gcs_file_store_bucket_name: indexer-grpc-file-store-bucketname
     data_service_grpc_tls_config:
+      data_service_grpc_listen_address: 0.0.0.0:50055
```

Comment:
wouldn't it be better to use the same addresses in the example that we use internally?

Reply:
updated; the plan is to use tls at 50052 and non-tls 50051.